oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/search/SearchAppLayout.jsx

```diff
@@ -1,26 +1,27 @@
-import React, { useContext } from "react";
+import React from "react";
 import PropTypes from "prop-types";
 import _isEmpty from "lodash/isEmpty";
 import Overridable from "react-overridable";
-import { withState, ActiveFilters } from "react-searchkit";
+import { withState, ActiveFilters, buildUID } from "react-searchkit";
 import { GridResponsiveSidebarColumn } from "react-invenio-forms";
 import { Container, Grid, Button } from "semantic-ui-react";
 import { i18next } from "@translations/oarepo_ui/i18next";
 import {
   SearchAppFacets,
   SearchAppResultsPane,
   SearchBar,
-  SearchConfigurationContext,
 } from "@js/invenio_search_ui/components";
 import { ResultOptions } from "@js/invenio_search_ui/components/Results";
 import { ClearFiltersButton } from "./ClearFiltersButton";
+import { parseSearchAppConfigs } from "@js/oarepo_ui";
 
 const ResultOptionsWithState = withState(ResultOptions);
 
 export const SearchAppLayout = ({ config, hasButtonSidebar }) => {
   const [sidebarVisible, setSidebarVisible] = React.useState(false);
-  const { appName, buildUID } = useContext(SearchConfigurationContext);
   const facetsAvailable = !_isEmpty(config.aggs);
+  const [searchAppConfig, ...otherSearchAppConfigs] = parseSearchAppConfigs();
```

Comment:
Here we shouldn't rely on this utility (here we already are inside a specific search app, which already has its specific configuration provided through its SearchConfigurationContext). You can fetch overridableIdPrefix from that context

Reply:
@mirekys Clear, thanks. So When I am inside of components I still use app name as before if I understand well?

Reply:
Yep it's fine to keep using appName as it's basically this: https://github.com/oarepo/oarepo-ui/blob/main/oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/search/util.js#L83

Reply:
@mirekys Clear thanks